### PR TITLE
cluster-status: updates to current requests

### PR DESCRIPTION
**Pull request by tyacbovi**

### CODE REVIEW

File: content/_cluster.md [hunk 1]
```diff
@@ -161,3 +161,335 @@ Only admin users can execute this operation.
...
+    headers=headers,
+    params=querystring,
+)
+response.json()
+```
+
+> Response Example
+
+```json
+{
```
Comment (inbala) [inline content/_cluster.md]: Maybe the response example should be shorter

File: content/_cluster.md [hunk 2]
```diff
@@ -161,3 +161,335 @@ Only admin users can execute this operation.
...
+```
+
+```python
+# Using ClodifyManager
+client.cluster_status.get_status()
+
+# Using requests
+url = 'http://<manager-ip>/api/v3.1/cluster-status'
+headers = {'Tenant': '<manager-tenant>'}
+querystring = {'_include': 'status'}
```
Comment (inbala) [inline content/_cluster.md]: Why do we need querystring?

File: content/_cluster.md [hunk 3]
```diff
@@ -161,3 +161,335 @@ Only admin users can execute this operation.
...
+
+### Getting summarised cluster status
+
+> Request Example
+
+```shell
+$ curl -X GET \
+    --header "Tenant: <manager-tenant>" \
+    -u <manager-username>:<manager-password> \
+    "http://<manager-ip>/api/v3.1/cluster-status"
```
Comment (inbala) [inline content/_cluster.md]: Where is the summary parameter?

File: content/_cluster.md [hunk 4]
```diff
@@ -161,3 +161,335 @@ Only admin users can execute this operation.
...
+)
+response.json()
+```
+
+> Response Example
+
+```json
+{
+  "status": "OK",
+  "services": {[]
```
Comment (inbala) [inline content/_cluster.md]: "services": {}

File: content/_cluster.md (hunk 3 above)
Comment (tyacbovi) [reply content/_cluster.md]: in the querystring

**Pull request merged**